core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -304,11 +317,13 @@ protected[core] object WhiskWebActionsApi extends Directives {
     findContentTypeInHeader(headers, transid, `text/html`).flatMap { mediaType =>
       val ct = ContentType(mediaType, () => HttpCharsets.`UTF-8`)
       ct match {
-        case _: ContentType.Binary | ContentType(`application/json`, _) =>
-          // base64 encoded json response supported for legacy reasons
-          Try(Base64.getDecoder().decode(str)).map(HttpEntity(ct, _))
+        // base64 encoded json response supported for legacy reasons
```

Comment:
Comment seems wrong here.

Reply:
is this clearer?

```scala
// base64 encoded json will appear as non-binary but it is excluded here for legacy reasons
```

Reply:
is this clearer?

// base64 encoded json will appear as non-binary but it is excluded here for legacy reasons